CHANGELOG.md

```diff
@@ -7,6 +7,60 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.15.0] - May, 25, 2023
+
+### React
+
+#### Added
+
+- [Breadcrumb] New Breadcurb component
+- [Hero] New Hero component
+- [StepByStep] New Step-by-step component
+
+- [Tooltip] Tooltips for status labels
+- New UI icons
+
+#### Fixed
+
+- [Accordion] Fixed accordion with borders
+
```

Comment:
Here or more suitable place:
[StatusLabel] Marked default rectangular Status label deprecating. This will be removed in 3.0.0